src/txn/txn_ckpt.c

```diff
@@ -956,6 +962,31 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
 
     __checkpoint_verbose_track(session, "sync completed");
 
+    /* If the history store file exists on disk, update its statistic. */
```

Comment:
There is already a check for `hs_dhandle` use the same here instead of adding new logic to find the HS file.